concepts/05 UI Components/FileUploader/50 Customize FileUploader.md

```diff
@@ -0,0 +1,162 @@
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div class="file-uploader-block"></div>
+        <DxFileUploader drop-zone=".file-uploader-block" />
+    </template>
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### React
+
+    <!-- tab: App.tsx -->
+    export default function App() {
+        return (
+            <div class="file-uploader-block"></div>
+            <FileUploader dropZone=".file-uploader-block" />
+        )
+    }
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+---
+
+This tutorial places FileUploader inside the custom drop zone container:
+
+---
+
+##### jQuery
+
+    <!-- tab: index.html -->
+    <div class="file-uploader-block">
+        <div id="file-uploader"></div>
+    </div>
+
+    <!-- tab: index.js -->
+    $(() => {
+        $('#file-uploader').dxFileUploader({
+            dropZone: '.file-uploader-block',
+        })
+    })
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <div class="file-uploader-block">
+        <dx-file-uploader dropZone=".file-uploader-block"></dx-file-uploader>
+    </div>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div class="file-uploader-block">
+            <DxFileUploader drop-zone=".file-uploader-block" />
+        </div>
+    </template>
+
+##### React
+
+    <!-- tab: App.tsx -->
+    export default function App() {
+        return (
+            <div class="file-uploader-block">
+                <FileUploader dropZone=".file-uploader-block" />
+            </div>
+        )
+    }
+
+---
+
+To center FileUploader elements inside a custom drop zone, assign the following CSS styles to the `.dx-fileuploader-input-wrapper` selector:
+
+    <!-- tab: styles.css -->
+    .dx-fileuploader-input-wrapper {
+        position: relative;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+    }
+
+When users upload files, the component lists selected files in its container. To modify placement of this list, implement absolute positioning on the uploaded file container. Assign the following CSS styles to the `.dx-fileuploader-files-container` selector:
+
+    <!-- tab: styles.css -->
+    .dx-fileuploader-files-container {
+        position: absolute;
+        top: 70%;
+        left: 50%;
+        transform: translate(-50%, 0);
+        width: 35%;
+        height: 30%;
+        overflow-y: auto;
+    }
+
+In this tutorial, the uploaded file container is positioned below the FileUploader with the `top: 70%` style. The `overflow-y: auto` style adds a scroll bar to the uploaded file container when items overflow the element.
```

Comment:
I usually try not to include pieces of code in the middle of the sentence, especially when those fragments are in the middle of the phrase - like "The `overflow-y: auto` style". It's just something you can't read comfortably. Since you list a couple of different styles here and explain their purpose, I would use a list. 

```suggestion
Note he following styles: 

- `top: 70%`: Positions the uploaded file list container below the FileUploader.
- `overflow-y: auto`: Adds a scroll bar to the uploaded file list container when necessary. 
```